es/ledger_serializer.go

```diff
@@ -32,6 +35,62 @@ func SerializeLedger(ledgerRow db.LedgerHeaderRow, transactionRows []db.TxHistor
 	return serializer.serialize()
 }
 
+// SerializeLedger serializes ledger data into ES bulk index data
+func SerializeLedgerFromHistory(networkPassphrase string, meta xdr.LedgerCloseMeta, buffer *bytes.Buffer) {
+	ledgerHeader := meta.V0.LedgerHeader.Header
+
+	ledgerRow := db.LedgerHeaderRow{
+		Hash:           hex.EncodeToString(meta.V0.LedgerHeader.Hash[:]),
+		PrevHash:       hex.EncodeToString(ledgerHeader.PreviousLedgerHash[:]),
+		BucketListHash: hex.EncodeToString(ledgerHeader.BucketListHash[:]),
+		LedgerSeq:      int(ledgerHeader.LedgerSeq),
+		CloseTime:      int64(ledgerHeader.ScpValue.CloseTime),
+		Data:           ledgerHeader,
+	}
+
+	transactionRows := make([]db.TxHistoryRow, len(meta.V0.TxSet.Txs))
+	feeRows := make([]db.TxFeeHistoryRow, len(meta.V0.TxSet.Txs))
+
+	for i, txe := range meta.V0.TxSet.Txs {
+		txHash, hashErr := util.HashTransactionInEnvelope(txe, networkPassphrase)
```

Comment:
@charlie-wasp looks like you forgot to include `util.HashTransactionInEnvelope`:

```
 go build -o dist/astrologer -v
github.com/astroband/astrologer/es
# github.com/astroband/astrologer/es
es/ledger_serializer.go:55:22: undefined: util.HashTransactionInEnvelope
```

Reply:
It's weird, I include `util` package [here](https://github.com/astroband/astrologer/pull/23/files#diff-14740c061c0ce9b871df002e2d3613eb49b51702af43dee23c03d97670b4a67fR10). I don't have any build errors on my dev machine, I'll try to investigate this

Reply:
@charlie-wasp yep, but there seems to be no `HashTransactionInEnvelope` method in [github.com/astroband/astrologer/util](https://github.com/astroband/astrologer/tree/play/stellar-core-replay/util) package.